index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>CarLotSale</title>
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div class="container">
+                <h1>Luxury Used Cars</h1>
+            </div>
+        </header>
+            <div class="navbar navbar-light bg-faded">
+                <input type="text" name="carSpecToChange" id="inputField" class="form-text">
+                <button id="doneButton" class="btn btn-success">DONE</button>
+            </div>
+        <div id="carCards" class="container">
+            <div class="row">
```

Comment:
This row should not be hard coded but rather dynamically created by javascript. What would happen if you added more cars? Would each set of three end up in their own rows?